.github/workflows/docs.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v5
       with:
-        python-version: '3.8'
+        python-version: '3.12'
 
     # This can be removed when `ibm-platform-services` can build/install with the most recent of
     # `setuptools`. `setuptools==72.0.0` and `ibm-platform-services==0.55.1` are a known-bad combo.
```

Comment:
Looks like there's another of these bits hanging out below the diff?

Reply:
oops, I'll remove it right now

Reply:
Done in: https://github.com/Qiskit/qiskit-neko/pull/51/commits/9bcf1137e13354140e9f0ce6b2e64ad3aaa6f027